Dockerfile

```diff
@@ -2,8 +2,8 @@
 ARG base_image=public.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base-nonroot:2021-12-01-1638322424
 # Build the manager binary
 # TODO(vijtrip2) move this builder image to public.ecr.aws/eks-distro-build-tooling/builder-base, when builder-base
-# supports golang 1.17
-FROM public.ecr.aws/bitnami/golang:1.17 as builder
```

Comment:
is bitnami image causing any issues? i think we should still keep the original image `public.ecr.aws/bitnami/golang`

Reply:
The `public.ecr.aws/bitnami/golang` does not support aarch64. The `public.ecr.aws/bitnami/golang:1.18` supports both (x86 and aarch64). I used golang 1.18 because of https://github.com/aws-controllers-k8s/community/issues/1438